src/main/java/io/github/hypixel_api_wrapper/wrapper/games/bedwars/HypixelBedWarsStats.java

```diff
@@ -4,52 +4,110 @@
 import io.github.hypixel_api_wrapper.wrapper.games.HypixelGame;
 import org.json.JSONObject;
 
+/**
+ * Used to retrieve information about a players BedWars stats. The methods in this class fetch
+ * information about a players statistics that apply to all of BedWars, not a specific mode. It can
+ * also be used to retrieve data about sub-modes of BedWars.
+ */
 public class HypixelBedWarsStats extends HypixelGame implements HypixelLootChestGame {
+
     public HypixelBedWarsStats(JSONObject stats) {
         super(stats);
     }
+
     @Override
     public String getGameID() {
         return "BEDWARS";
     }
+
     @Override
     public String getGameName() {
         return "Bed Wars";
     }
+
     @Override
     public boolean isRemoved() {
         return false;
     }
 
-
+    /**
+     * Instantiates and returns a {@link HypixelBedWars}.
+     *
+     * @param mode The sub-mode of BedWars that the {@link HypixelBedWars} will reference when
+     *             retrieving information.
+     * @return A {@link HypixelBedWars} that holds a {@link JSONObject} and
+     * {@link HypixelBedWarsMode} that will be used to fetch data from the API.
+     */
     public HypixelBedWars getMode(HypixelBedWarsMode mode) {
         return new HypixelBedWars(stats, mode);
     }
 
+    /**
+     * Coins can be added coins to a players profile from kills, bed-breaks, quest
+     * completions, wins, and more. They can be used to purchase BedWars cosmetics.
+     *
+     * @return The amount of BedWars coins a player has.
+     */
```

Comment:
Add an apostrophe to "player's", and put a space between "Bed" and "Wars"
```suggestion
    /**
     * Coins are earned by winning, killing other players, breaking beds, and completing quests.
     * They can be used to purchase Bed Wars cosmetics, such as shopkeeper skins and victory dances.
     *
     * @return The total number of Bed Wars coins the player currently has.
     */
```